server/graphql/v2/query/RequiredBankInformationQuery.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+
+import RequiredBankInformation from '../object/RequiredBankInformation';
+import models from '../../../models';
+
+const RequiredBankInformationQuery = {
```

Comment:
I would not do that as a top-level query. If this is a property of the host, it should be at property on the `Account` level, perhaps on a dedicated `Host` object.

Reply:
Moved to Account so we can update the existing query to
```gql
host {
  transferwise {
    availableCurrencies
    requiredFields(currency: "EUR")
  }
}
```